pygmt/tests/test_session_management.py

```diff
@@ -57,3 +58,34 @@ def test_gmt_compat_6_is_applied(capsys):
         # Make sure no global "gmt.conf" in the current directory
         assert not os.path.exists("gmt.conf")
         begin()  # Restart the global session
+
+
+def _gmt_func_wrapper(figname):
+    """
+    A wrapper for running PyGMT scripts with multiprocessing.
+
+    Currently, we have to import pygmt and reload it in each process. Workaround from
+    https://github.com/GenericMappingTools/pygmt/issues/217#issuecomment-754774875.
+    """
+    from importlib import reload
```

Comment:
Can move this import to the top?

Reply:
Done in 236f874c44daec321d5ce90e54fe1dabe4ba7071